playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -4,14 +4,22 @@
 from collections import namedtuple
 
 
-def is_oom_status(status: ContainerStatus):
+def is_last_state_in_oom_status(status: ContainerStatus):
     if not status.lastState:
         return False
     if not status.lastState.terminated:
         return False
     return status.lastState.terminated.reason == "OOMKilled"
 
 
+def is_state_in_oom_status(status: ContainerStatus):
```

Comment:
can we avoid the code duplication?

Reply:
We can, but I think it makes the code less readable. Are you okay with avoiding this change?